packages/node_modules/@webex/internal-plugin-device/test/integration/spec/device.js

```diff
@@ -67,135 +67,64 @@ describe('plugin-device', () => {
     });
 
     describe('#canRegister()', () => {
-      describe('when wait is truthy', () => {
-        let wait;
-
-        beforeEach('set wait', () => {
-          wait = true;
-        });
-
-        describe('when the `wdm` service is available', () => {
-          let services;
-
-          beforeEach('destructure services plugin and get catalog', () => {
-            services = webex.internal.services;
-
-            return services.waitForCatalog('postauth')
-              .then(() => {
-                assert.isDefined(services.get('wdm'));
-              });
-          });
-
-          it('should return a resolved promise', () => {
-            assert.isFulfilled(device.canRegister(wait));
-          });
-        });
-
-        describe('when the service catalog is not ready', () => {
-          let services;
+      describe('when the `wdm` service is available', () => {
+        let services;
 
-          beforeEach('setup catalog to be not ready', () => {
-            services = webex.internal.services;
+        beforeEach('destructure services plugin and get catalog', () => {
+          services = webex.internal.services;
 
-            services.updateServices();
-          });
-
-          describe('when the \'wdm\' service does exist after wait', () => {
-            it('should return a resolved promise', () => {
-              /* eslint-disable-next-line no-underscore-dangle */
-              assert.isTrue(services._getCatalog().status.postauth.collecting);
-              assert.isFulfilled(device.canRegister(wait));
-            });
-          });
-
-          describe('when the \'wdm\' service does not exist after wait', () => {
-            beforeEach('remove wdm service', () => {
-              services.get = sinon.stub().returns(undefined);
-              services.waitForCatalog = sinon.stub().resolves();
-            });
-
-            it('should return a rejected promise', () => {
-              /* eslint-disable-next-line no-underscore-dangle */
-              assert.isTrue(services._getCatalog().status.postauth.collecting);
-              assert.isRejected(device.canRegister(wait));
+          return services.waitForCatalog('postauth')
+            .then(() => {
+              assert.isDefined(services.get('wdm'));
```

Comment:
Can you remove this assert that lives in the `beforeEach` block?

Reply:
implemented.